client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -667,20 +669,120 @@ class ContentDataProvider
     if (!targetUri) {
       return false;
     }
-
     const closing = closeFileIfOpen(item);
-    if (!(await closing)) {
+    const closedFiles = await closing;
+    if (!closedFiles) {
       return false;
     }
 
     const newUri = await this.model.moveTo(item, targetUri);
-    if (closing !== true) {
-      commands.executeCommand("vscode.open", newUri);
+    if (Array.isArray(closedFiles) && closedFiles.length > 0) {
```

Comment:
With drag and drop, at the moment, we call `moveItem` for each item dragged, and we call `closeFileIfOpen` for _one_ item. And, `newUri` _should_ be returned after the item is moved.

Is there an issue where the `newUri` is wrong? If _that's_ the case, this should be happening at the model level